model/src/main/kotlin/config/HttpFileStorageConfiguration.kt

```diff
@@ -28,6 +29,15 @@ data class HttpFileStorageConfiguration(
      */
     val url: String,
 
+    /**
+     * The postfix that is appended to the combination of the url and some additional path. Some storages process
+     * authentication via parameters that are within the final URL, so certain credentials can be stored in this
+     * postfix, e.g, "?user=standard&pwd=123". Thus, the final URL could be
+     * "https://example.com/storage/path?user=standard&pwd=123".
+     */
+    @JsonProperty("urlPostfix")
```

Comment:
Keep the default snake case like in other configuration classes. Instead, consider not serializing this property if it is empty.